pipelines/smartseq2_by_plate/SmartSeq2Plate.wdl

```diff
@@ -0,0 +1,133 @@
+import "SmartSeq2SingleSample.wdl" as single_cell_run
+
+task AggregateDataMatrix{
+  Array[File] filename_array
+  String col_name
+  String output_name
+  String docker = "gcr.io/broad-dsde-mint-dev/benchmarking-tools:0.0.1"
+  command {
+    set -e
+    gsutil cp gs://broad-dsde-mint-dev-teststorage/pipeline_testing_scripts/MergeDataMatrix.py ./
```

Comment:
This seems a little brittle. I think someone would be surprised that deleting or moving a script in a google bucket would break other WDLs.

Reply:
I agree - 
I'm curious why was a hard-coded String instead of a `File` input? It feels like something we should try to address in the spec if WDL encourages (or forces!) hard-coded file paths